frontend/components/Dashboard/Nav/NavLinks.tsx

```diff
@@ -153,16 +152,16 @@ const NavLinks: React.FC<Props> = ({ onClick, currentUser }) => {
             margin-top: 15px;
             font-size: 16px;
             text-align: center;
-            color: white;
+            color: ${theme.colors.white};
             animation: fadeIn ${navConstants.transitionDuration}ms linear;
           }
         }
 
         .nav-bottom {
           display: flex;
           flex-direction: column;
-          /* Expands the links to push the Journaly "J" logo down to the bottom */
-          flex-grow: 2;
+          /* centers itself in the grid container */
```

Comment:
The container for the nav links is centered vertically with grid already, but this makes the links within the container centered in the container